package.json

```diff
@@ -74,7 +74,9 @@
     "magic-wand-js": "1.0.0",
     "moment": "2.23.0",
     "numeral": "1.5.3",
-    "select2": "4.0.3",
+    "react": "16.3.2",
+    "react-dom": "16.3.2",
```

Comment:
Are react and react-dom part if this PR?

Reply:
Nope - I didn't bring that in

Reply:
Still unclear why react and react-dom are here (those are getting added in this PR; ie. git doesn't lie :) )